backend/overstroomik_service/pdok.py

```diff
@@ -63,20 +61,25 @@ async def fetch_data(self, url: str, params: dict):
         :param url: api url to fetch
         """
         address_item = {}
-        
+
         # connect async to the pdok-api
         async with httpx.AsyncClient() as client:
 
             # fetch the feature info
             try:
-                result = await client.get(url=url, params=params, timeout=settings.FETCH_TIMEOUT)
-                if result.status_code == httpx.codes.OK:                    
+                result = await client.get(
+                    url=url, params=params, timeout=settings.FETCH_TIMEOUT
+                )
+                if result.status_code == httpx.codes.OK:
                     status, address_item = self.list_to_location(result.json())
-                else:                    
+                else:
                     status = Errors.ERROR_PDOK_NO_RESP
 
-            except:                
```

Comment:
Never do a bare `except` as it will catch absolutely everything (including things like ctrl-c, end of the world, etc). In this case we also didn't log any information about why it was failing, which makes debugging (in the case of getting communication within Kubernetes to work) quite hard.